src/nethsec/firewall/__init__.py

```diff
@@ -1191,6 +1191,9 @@ def is_zone(uci, name:str) -> bool:
     for value in zones.values():
         if 'name' in value and value['name'] == name:
             return True
+        # True zone if name is 'any' zone name -> '*'
+        elif name == '*':
+            return True
```

Comment:
This should not be inside the loop. It's enough to check it once at beginning of the function.